pnc-core/src/main/java/org/jboss/pnc/core/builder/BuildCoordinator.java

```diff
@@ -1,23 +1,5 @@
 package org.jboss.pnc.core.builder;
```

Comment:
I don't think that commiting a file without meaningful changes is a good practice. It adds only a noise.